src/EETH.sol

```diff
@@ -139,8 +153,13 @@ contract EETH is IERC20Upgradeable, UUPSUpgradeable, OwnableUpgradeable, IERC20P
         _approve(owner, spender, value);
     }
 
-    // [INTERNAL FUNCTIONS] 
+    //--------------------------------------------------------------------------------------
+    //-------------------------------  INTERNAL FUNCTIONS  ---------------------------------
+    //--------------------------------------------------------------------------------------
+
     function _transfer(address _sender, address _recipient, uint256 _amount) internal {
+        require(blacklistedRecipient[_recipient] == false, "eETH: recipient blacklisted"); 
```

Comment:
According to the spec, the check happens  both on sender and recipient; 

```
require(!blacklisted[_sender] || !blacklisted[_recipient], "EETH: blacklisted address");
```

Reply:
Q2) how much gas cost does it add?

Reply:
Initial Gas Cost: 49948
Gas Cost with check only on `to`: 52161
Gas Cost if we add checks on both `to` and `from`: 54382

Reply:
thx that is good to know.

2100 is due to the cold SLOAD :)
https://hackmd.io/@fvictorio/gas-costs-after-berlin#SLOAD-after-Berlin